lib/services/auth.dart

```diff
@@ -65,6 +64,12 @@ class AuthService {
     }
   }
 
+  // forgot password
+
+  Future<void> resetPassword(String email) async {
+    await _auth.sendPasswordResetEmail(email: email);
```

Comment:
I don't think this needs to be async since sendPasswordResetEmail already returns a Future. Try to remove the async and await from here.

Reply:
Yeah it works without them. Changed.

Reply:
See this too..